### DISCUSSION TITLE
"@umijs/max"”没有导出的成员“MicroAppLink”

### DISCUSSION THREAD

**Discussion by ZFRong** [umijs/umi#10659] [Q&A]
<img width="731" alt="WeChat3a0e761f8e9ac23f70555b5d93bb1cc3" src="https://user-images.githubusercontent.com/20680975/222374507-9edf0bc0-8fb8-4eb9-b921-30dcea63cefd.png">
<img width="778" alt="WeChat6b09f6b6de5b801c34eceeaac51d1dce" src="https://user-images.githubusercontent.com/20680975/222374717-bb04710d-cc25-45e8-bc97-2b450b5c9241.png">

**Comment by fz6m** [comment]
开了 `qiankun.slave` 才有它。

**Comment by ZFRong** [comment]
好的，谢谢



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2023年3月2日(周四) 晚上10:00
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [umijs/umi] ***@***.***/max"”没有导出的成员“MicroAppLink” (Discussion #10659)




 
开了 qiankun.slave 才有它。
